### DISCUSSION TITLE
Copilot usage in russia

### DISCUSSION THREAD

**Discussion by keetsta** [community/community#83138] [Copilot Conversations] [answered]
I have access to copilot under a student license, but at the moment it is blocked in Russia. Are there any ways to use it without vpn or similar?

**Comment by faridnec** [comment, marked as answer]
I am sorry to hear that, It is either VPN or proxy services you can use (though, keep in mind that using tools to circumvent restrictions might violate the terms of service of the platform and could have legal implications). Also alternatively you can reach out to GitHub Support. They may provide information on any restrictions and potential solutions.
